src/data/LapisResponse.ts

```diff
@@ -7,6 +7,6 @@ export type LapisInformation = {
 
 export type LapisResponse<T> = {
```

Comment:
Since this is a generic, this was hard to do. But I added a function, which returns a schema from the generic. This is now used to parse the responses.